### DISCUSSION TITLE
Helm mixin install chart from local

### DISCUSSION THREAD

**Discussion by dejanu** [getporter/porter#3234] [Q&A] [answered]
I'm using the helm mixin, but currently I'm unsure if it the `install` actions  supports installing a chart from local (from tarball) , e.g. `helm install  ... ./nginx-1.2.3.tgz`?  Thx.

**Comment by kichristensen** [comment, marked as answer]
I don't see why it shouldn't be able to, but it would be better to ask to the maintainer of the Helm mixin over here, https://github.com/MChorfa/porter-helm3

**Reply by dejanu** [reply]
Yep it works 😅
